src/api/app/assets/javascripts/webui2/project_monitor.js

```diff
@@ -6,18 +6,75 @@ function setAllLinks(event) {
   $(this).closest('.dropdown-menu').find('input').prop('checked', event.data.checked);
 }
 
-function setupProjectMonitor() { // jshint ignore:line
+function statusCell(meta, statusHash, tableInfo, projectName, packageName) {
+  var info = tableInfo[meta.col - 1];
+  var repository = info[0];
+  var architecture = info[1];
+  var status = statusHash[repository][architecture][packageName] || {};
+  var code = status.code;
+  if (code === undefined) { return null; }
+
+  var output = '<a ';
+  if (['succeeded', 'failed', 'building'].includes(code)) {
+    var url = '/package/live_build_log/' + projectName + '/' + packageName + '/' + repository + '/' + architecture;
+    output += 'href="' + url + '"';
+  }
+  else {
+    var id = meta.row + '-' + meta.col;
+    output += 'href="javascript:void(0);" id="' + id + '"';
+
+    if (status.details !== undefined) {
+      if (code === 'scheduled') klass = 'text-warning';
```

Comment:
Please wrap this in `{}` like you did [above](https://github.com/openSUSE/open-build-service/pull/6981/files#diff-52aa359322a395d7dce7d088fe03e479R15)

Reply:
or do not wrap it above, as it is not needed....

Reply:
Is this `klass` variable ever used? Later in line 30 it is redefined.